src/components/JobListings/index.jsx

```diff
@@ -2,21 +2,21 @@ import Header from "../Header/index.jsx";
 import Table from "../Table/index.jsx";
 import {useEffect, useState} from "react";
 
-const JobListings = () => {
-    const [headerText, setHeaderText] = useState('Most recent jobs')
-    const [jobInfo, setJobInfo] = useState([])
+const JobListings = ({setQuery, query, setHeader, header, viewButton, setViewButton}) => {
+
+    const [recentJobInfo, setRecentJobInfo] = useState([])
     useEffect(() => {
-        const getJobInfo = async () => {
-            const response = await fetch('http://0.0.0.0:8080/jobs/recent')
+        const getRecentJobInfo = async () => {
```

Comment:
Should this really be called `recentJobInfo`, when it will sometimes be the result of a search query that includes other things too?

Reply:
I see why you've done this: the default is the recent job info, but don't forget that this same function is used to re-fetch other information too, so this name is misleading.